gdbinit.py

```diff
@@ -19,39 +19,55 @@
 virtual_env = environ.get("VIRTUAL_ENV")
 
 if virtual_env:
-    no_venv_warning = environ.get("PWNDBG_NO_VENV_WARNING", 0)
-    if not no_venv_warning and not sys.executable.startswith(virtual_env):
-        print("[!] Pwndbg Python virtualenv warning [!]")
-        print(
-            "Found Python virtual environment (VIRTUAL_ENV='%s') while GDB is built with a different Python binary (%s)"
-            % (virtual_env, sys.executable)
-        )
-        print("Assuming that you installed Pwndbg dependencies into the virtual environment")
-        print("If this is not true, this may cause import errors or other issues in Pwndbg")
-        print(
-            "If all works for you, you can suppress this warning by setting PWNDBG_NO_VENV_WARNING=1"
-        )
-        print("")
+    venv_warning = int(environ.get("PWNDBG_VENV_WARNING", 1))
+    py_exe_matches = sys.executable.startswith(virtual_env)
+
+    if not py_exe_matches:
+        if venv_warning:
+            print("WARNING: Pwndbg/GDB run in virtualenv with which it may not work correctly ***")
```

Comment:
Just a suggestion: might be better to make this a multiline f-literal string instead of a bunch of prints.